<launch>

  <rosparam file="$(find rm_base)/config/actuator_coefficient.yaml" command="load" ns="rm_base"/>
  <rosparam file="$(find rm_base)/test/config/test_3508.yaml" command="load"/>

  <rosparam file="$(find rm_base)/test/urdf/test_3508.xacro" command="load" ns="robot_description"/>

  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" args="
        rm_base/robot_state_controller
        rm_base/joint1_position_controller
"/>

  <!--  <node name="rm_base" pkg="rm_base" type="rm_base" respawn="false"/>-->

</launch>